editor/src/clj/editor/command_requests.clj

```diff
@@ -91,6 +91,10 @@
    {:ui-handler :engine-profile-show
     :help "Open the Engine Profiler in a web browser."}
 
+   :engine-resource-profiler
+   {:ui-handler :engine-profile-show
+    :help "Open the Engine Resource Profiler in a web browser."}
+
```

Comment:
Shouldn't the `:ui-handler` be `:engine-resource-profile-show` here?

Reply:
thx